src/scss/components/_ratings.scss

```diff
@@ -0,0 +1,63 @@
+$star-count: 6;
+$star-size-small: 14px;
+$star-size-medium: 28px;
+$star-size-large: 42px;
+
+.c-ratings {
+    background: inline('#{$icon-path}/stars/star--empty.svg') repeat-x;
+    background-size: contain;
+    display: inline-block;
+    height: $star-size-small;
+    width: $star-size-small * $star-count;
+}
+
+.c-ratings--medium {
+    height: $star-size-medium;
+    width: $star-size-medium * $star-count;
+}
+
+.c-ratings--large {
+    height: $star-size-large;
+    width: $star-size-large * $star-count;
+}
+
+.c-ratings--fill {
+    background: inline('#{$icon-path}/stars/star--filled.svg') repeat-x;
+    background-size: contain;
+    display: inline-block;
+    height: 100%;
+    width: 100%;
+    @include rating-fill($star-count);
+}
+
+/*
+Selectable stars rating component
+*/
+.c-ratingSelectable {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+}
+
+.c-ratingSelectable-input {
+    display: none;
+
+    &:checked {
+        & + label,
+        & ~ input + label {
+           background-image: inline('#{$icon-path}/stars/star--filled.svg');
+        }
+   }
+}
+
+.c-icon--star {
+    display: block;
+    cursor: pointer;
+    background-image: inline('#{$icon-path}/stars/star--empty.svg');
```

Comment:
Same as above - would be good to use a silent extends class for this to help with maintenance.